index.html

```diff
@@ -555,7 +556,102 @@ <h2>Dictionary {{RTCRtpEncodingParameters}} Members</h2>
         allows the <a>user agent</a> to adapt its bandwidth allocation strategy
         based on the current network conditions.</p>
       </dd>
+      <dt><dfn data-idl>codec</dfn> of type <span class="idlMemberType">RTCRtpCodec</span></dt>
+      <dd>
+        <p>Optional value overriding the codec used for this encoding. The {{RTCRtpCodec}} dictionary is
+          defined in [[WEBRTC]]. If absent, the default codec selection algorithm selects the first
+          negotiated codec.</p>
+          <div class="issue atrisk">
+            <p>
+              {{RTCRtpEncodingParameters/codec}} is marked as a feature at risk
```

Comment:
I would not mark it as a feature at risk, maybe mention that it is a new feature asking fro implementers feedback

Reply:
marking a feature "at risk" only makes sense in the context of a Candidate Recommendation; since that document is not intended to be published on the Rec track, but instead to feed into the WebRTC Rec (AFAIK), then I don't think marking anything at risk serves much purpose indeed.

Reply:
The "feature at risk" designation is there to keep track of what has been implemented (and is therefore eligible to be added to WebRTC-PC).

Reply:
Feature at risk has been removed. We can add it later if implementations are lacking.